tools/releasing/create_source_release.sh

```diff
@@ -17,6 +17,9 @@
 # limitations under the License.
 #
 
+## check required Java and Maven version
+. check_java_maven_versions.sh
```

Comment:
I would still object adding this check here because we're not using Java or Maven in this script. We could even remove the line below where the `MVN` variable is defined.